assets/lib/font-awesome/update-fontawesome.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Move it to a folder that will be removed on build.
Please consult with @KingYes
Currently, this file will be accessible from the web.